{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "cb724cba_3356bcc1",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1018946
      },
      "writtenOn": "2023-01-17T16:41:26Z",
      "side": 1,
      "message": "It was tested downstream and with this change undercloud update completes without any error. ",
      "revId": "164ca7c4b7d4e9263b525f4e25dde1d36060bdcb",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}